src/backend/cuda/sparse.cu

```diff
@@ -145,6 +145,21 @@ struct nnz_func_def_t
                                               int *, int *);
 };
 
+//cusparseStatus_t cusparseZgthr(cusparseHandle_t handle,
```

Comment:
This is in line with other libraries/code we are using where we keep a commented version of the function declaration around as reference.